proposals/description/fip_22.json

```diff
@@ -0,0 +1,66 @@
+{
+    "proposal_title": "FIP-22: Deploy FEI liquidity and TRIBE incentives to Aave",
+    "proposal_commands": [
+        {
+            "address": "0x956F47F50A910163D8BF957Cf5846D573E7f87CA",
+            "target" : "fei",
+            "values": "0",
+            "method": "mint(address,uint256)",
+            "arguments": [
+                "0xB51f09B6F103D697dc5d64DC904Ad6a2Dad39987",
+                "25000000000000000000000000"
+            ],
+            "description": "Grant Burner to Eth Reserve Stabilizer"
+        },
+        {
+            "address": "0xB51f09B6F103D697dc5d64DC904Ad6a2Dad39987",
+            "target" : "rariPool6FeiPCVDeposit",
```

Comment:
note this key and address 0xB51 are placeholders until the actual FEI deposit is deployed